tests/standAloneClient/bpmNotificationTemplatesForBurntSubtitlesAdditionalTriggers.xml

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why not add this into bpmNotificationTemplatesForBurntSubtitles.xml and just re-run it again?